pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,38 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
+accessibility_requirements:
+  wcag20:x.x.x: # (Name of Success Criterion)
+    forConformance: true
```

Comment:
shouldn't it be 
true | false

Reply:
@carlosapaduarte, I guess the point here is that success criteria can only be listed here, if failing the rule means failing the succes criterion, and in that case the rule will always be required for conformance. 

If it's not required for conformance, a technique could possibly be listed as the accessibility requirement as now shown on line 22.